client/emv/cmdemv.c

```diff
@@ -294,19 +296,23 @@ int UsageCmdHFEMVExec(void) {
 }
 
 #define TLV_ADD(tag, value)( tlvdb_add(tlvRoot, tlvdb_fixed(tag, sizeof(value) - 1, (const unsigned char *)value)) )
+#define dreturn(n) do{DropField();return n;}while(0)
```

Comment:
do ... while(0)? Doesn't it work without?

Reply:
I belive @pwpiwi  means this:

 #define dreturn()  { DropField(); }

Reply:
works. )   cool)